### ISSUE TITLE
Choice of F.grid_sample align_corners flag due to different pytorch version

### ISSUE THREAD

**Issue by ZetongZhangZ** [TWJianNuo/EdgeDepth-Release#12]
Hi,

Thank you for the helpful code! I have a question regarding F.grid_sample used in your code. When I run it, there is a warning: "UserWarning: Default grid_sample and affine_grid behavior has changed to align_corners=False since 1.3.0. Please specify align_corners=True if the old behavior is desired. See the documentation of grid_sample for details.". Since the pytorch version is not specified in the README, may I ask what is the environment of your experiment? I am now running it with pytorch 1.7.0, where the default align_corners=False. Is it the same as your case? Thank you in advance!

**Comment by TWJianNuo** [comment]
Yes. By default, I use the falg set as False. Though I believe it makes minor difference between the two.
